internal/credential/vault/repository_credential_store.go

```diff
@@ -121,6 +122,14 @@ func (r *Repository) CreateCredentialStore(ctx context.Context, cs *CredentialSt
 		return nil, errors.Wrap(ctx, fmt.Errorf("scheduler interval must be greater than token ttl. scheduler jobs interval value: %s", runJobsInterval.String()), op)
 	}
 
+	period, err := parseutil.ParseDurationSecond(tokenLookup.Data["period"])
```

Comment:
I think we are safe on the credential store with the previous PR that covers the token TTL because we always renew the token to the period when we create/update the token in the store.

This warning should be logged when a credential is issued from a credential library [here](https://github.com/hashicorp/boundary/blob/main/internal/credential/vault/repository_credentials.go#L41)